src/NServiceBus.Core/Hosting/ExternallyManagedContainerHost.cs

```diff
@@ -34,13 +34,13 @@ public ExternallyManagedContainerHost(EndpointCreator endpointCreator, HostingCo
 
         internal Lazy<IServiceProvider> Builder { get; private set; }
 
-        public async Task<IEndpointInstance> Start(IServiceProvider externalBuilder, CancellationToken cancellationToken = default)
+        public async Task<IEndpointInstance> Start(IServiceProvider serviceProvider, CancellationToken cancellationToken = default)
```

Comment:
I left the interface name as is since that would be a breaking change. I think it is ok to have here a different name. The previous code also called it `externalBuilder` while the interface calls it `builder`

Reply:
differences in names are also breaking changes as the code would no longer compile if a user would use named arguments.

Reply:
This is the only public argument so this is the only one that needs to be reverted.

Reply:
@ramonsmits this is our internal implementation https://github.com/Particular/NServiceBus/blob/6d2e1f89201ae8d491e5b605c44ebc2ef62c145e/src/NServiceBus.Core/Hosting/ExternallyManagedContainerHost.cs#L7 of the interface. So the public is not public

Reply:
Sorry, I expanded, saw public at the top but that was part of the constructor🤦